model/kubernetes/kubernetes_cluster.rb

```diff
@@ -8,6 +8,7 @@ class KubernetesCluster < Sequel::Model
   many_to_one :private_subnet
   many_to_one :project
   many_to_many :cp_vms, join_table: :kubernetes_clusters_cp_vms, class: :Vm, order: :created_at
+  one_to_many :nodepools, key: :kubernetes_cluster_id, class: :KubernetesNodepool
```

Comment:
Is `key` needed here?  The given value looks the same as what Sequel would use by default.